src/kOS.Safe/Screen/TextEditor.cs

```diff
@@ -1,6 +1,7 @@
 using System.Collections.Generic;
 using System.Text;
 using kOS.Safe.Utilities;
+using System;
 
 namespace kOS.Safe.Screen
 {
```

Comment:
The weird thing to remember is that TextEditor, despite the name, is really only used now as the base for the commandline editing logic of the interactive interpreter.